### ISSUE TITLE
Update of history

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-testsuitesmanagement#113]
The latest entry in history within the documentation is

```
0.3.0 07/2022
Added local configuration feature; documentation rework
```

This is rather old and in between lots of maintenance activities happened.

It makes probably no sense to list every single maintenance task in history (because this would overload the history), but for the
upcoming release of the RobotFramework AIO I suggest, at least to increase version to 0.4.0 and mention in history something common like

`Maintenance of log output; maintenance of computation of configuration files`

**Comment by test-fullautomation** [comment]
Hi Son,
can you please provide a corresponding pull-request?
I plan also to update pipy with this.
Thank you, 
Thomas

**Comment by namsonx** [comment]
Hello Thomas,

I created the pull-request #135  for this history update.

Thank you,
Son

**Comment by test-fullautomation** [comment]
solved and merged with #135
